crates/ruma-events/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # [unreleased]
 
+Breaking changes:
+
+ - Add support for encrypted stickers as sent by several bridges under the flag `compat-encrypted-stickers`
```

Comment:
This could be listed as an improvement. The breaking change here is that the `url` field of `StickerEventContent` has been replaced with the `source` field that is a `StickerMediaSource`.